backend/src/utils/logging/logger.ts

```diff
@@ -1,15 +1,56 @@
-import pino from "pino";
-
-export const logger = pino({
-  level: process.env.PINO_LOG_LEVEL || "trace",
-  timestamp: pino.stdTimeFunctions.isoTime,
-  formatters: {
-    bindings: (bindings) => {
-        return {
+import pino, { Logger } from "pino";
+import { getAwsCloudWatchLog, getNodeEnv } from "../../config";
+
+export let logger: Logger;
+
+export const initLogger = async () => {
+  const awsCloudWatchLogCfg = await getAwsCloudWatchLog();
+  const nodeEnv = await getNodeEnv();
+  const isProduction = nodeEnv === "production";
+  const targets: pino.TransportMultiOptions["targets"][number][] = [
+    isProduction
+      ? { level: "trace", target: "pino/file", options: {} }
```

Comment:
I don't hink we don't need this low level of log level for prod. Info is detailed enough